### ISSUE TITLE
Implement view containing list of deploys

### ISSUE THREAD

**Issue by RitaMAllenCA** [casper-network/casper-blockexplorer#235]
@RitaMAllenCA commented on [Fri Jan 27 2023](https://github.com/casper-network/casper-blockexplorer-frontend-old/issues/2)

@RitaMAllenCA commented on [Fri Dec 09 2022](https://github.com/casper-network/casper-blockexplorer/issues/105)

looking for a list of deploys, rather than depending on the deploy hash for display

---

@hoffmannjan commented on [Fri Jan 27 2023](https://github.com/casper-network/casper-blockexplorer/issues/105#issuecomment-1406613089)

@RitaMAllenCA this is both `middleware` and `frontend` ticket. 
So this requires three tickets:
- Design view containing list of deploys (`frontend`) cc: @megan-demit 
- Implement view containing list of deploys (`frontend`)
- Implement middleware endpoint for `/deploys` (`middleware`)

---

@RitaMAllenCA commented on [Thu Feb 02 2023](https://github.com/casper-network/casper-blockexplorer-frontend-old/issues/2#issuecomment-1414516188)

@hoffmannjan This should be an epic with at least 3 tickets - one for design, one for FE and one for MW.  However I believe it is partially blocked by the endpoint being available in sidecar (we can only use the node endpoint until sidecar is available).  I don't want to break it out until we move it to the explorer repo.....

**Comment by RitaMAllenCA** [comment]
duplicate of #382
